backend/store/media.py

```
@@ -0,0 +1,57 @@
+        getattr(config, "STORE_BACKEND", "filesystem") == "appwrite"
+        and bool(config.APPWRITE_BUCKET_VOICE_NOTES)
+        and bool(config.APPWRITE_DATABASE_ID)
+    )
+
+
+def upload_audio_file(filename: str, data: bytes, mime: str) -> Optional[str]:
+    if not is_appwrite_storage_enabled():
+        return None
+    try:
+        return _client().upload_file(config.APPWRITE_BUCKET_VOICE_NOTES, filename, data, mime)
+    except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)